### PR TITLE
:sparkles:  add how it works section

### PR DESCRIPTION
**Pull request by MarioBatalha**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206109714345737

### CODE REVIEW

File: views/institutional/components/feature/feature-header.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    mt="2xl"
+    py={['2rem', '2rem', '2rem', '4rem']}
+    mx="auto"
+    width={['100%', '100%', '100%', '40rem']}
+    display="flex"
+    alignItems="center"
+    flexDirection="column"
+    justifyContent="center"
+  >
+    <Tag variant="outline" size="medium" width="max-content" mb="2rem">
```
Comment (atilasousa) [inline views/institutional/components/feature/feature-header.tsx]: check the ui kit for this mb size

File: views/institutional/components/feature/index.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import { Box } from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+
+import FeatureDescription from './feature-description';
+import FeatureHeader from './feature-header';
+
+const Feature: FC = () => (
+  <Box
+    height={['260vh', '260vh', '260vh', '305vh']}
+    width="100vw"
```
Comment (atilasousa) [inline views/institutional/components/feature/index.tsx]: pleasee, check if it's really necessary this width

File: views/institutional/components/feature/feature-header.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      How it works
+    </Tag>
+    <Typography
+      size="large"
+      mb="1rem"
```
Comment (atilasousa) [inline views/institutional/components/feature/feature-header.tsx]: check this size in the ui kit

File: views/institutional/components/feature/feature-description.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    justifyContent="center"
+    alignItems="center"
+  >
+    <Typography
+      textAlign="center"
+      as="h1"
+      whiteSpace="wrap"
+      fontSize={['2.5rem', '3.563rem']}
+      fontWeight="900"
+      lineHeight="4rem"
```
Comment (atilasousa) [inline views/institutional/components/feature/feature-description.tsx]: check this size in the ui kit

**Review by KipandaJr — CHANGES_REQUESTED** [review]
your approach of using the backgrounds as svg and then rendering them on the site isn't going to work in all cases, I suggest you think about it differently, or say something so we can sort it out in the `war-room` @MarioBatalha

**Comment by MarioBatalha** [comment]
> your approach of using the backgrounds as svg and then rendering them on the site isn't going to work in all cases, I suggest you think about it differently, or say something so we can sort it out in the `war-room` @MarioBatalha

Image in webp format reduce image quality

**Review by KipandaJr — CHANGES_REQUESTED** [review]
when u free tell me to do this on pair programming in war-room.

File: views/institutional/index.tsx [hunk 5]
```diff
@@ -2,11 +2,13 @@ import { FC } from 'react';
 
 import Layout from '@/components/layout';
 
+import Feature from './components/feature/index';
 import { Benefits, Hero } from './components';
```
Comment (git-marcopitra) [inline views/institutional/index.tsx]: export it on the Baril, to be like this other sections

File: views/institutional/components/feature/feature-header.tsx [hunk 6]
```diff
@@ -0,0 +1,39 @@
+import { Box, Tag, Typography } from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+
+const FeatureHeader: FC = () => (
+  <Box
+    mt="2xl"
+    py={['2rem', '2rem', '2rem', '4rem']}
```
Comment (git-marcopitra) [inline views/institutional/components/feature/feature-header.tsx]: change this values to theme keys

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

File: views/institutional/components/feature/feature-header.tsx [hunk 7]
```diff
@@ -0,0 +1,39 @@
+import { Box, Tag, Typography } from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+
+const FeatureHeader: FC = () => (
+  <Box
+    mt="2xl"
+    py={['3xl', '3xl', '3xl', '4rem']}
```
Comment (KipandaJr) [inline views/institutional/components/feature/feature-header.tsx]: `4rem` is `8xl`

File: views/institutional/components/feature/feature-header.tsx [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+    <Typography
+      size="large"
+      mb="m"
+      variant="title"
+      fontWeight="700"
+      fontStyle="normal"
+      textAlign="center"
+      fontSize="7xl"
+      lineHeight="5xl"
+      fontFamily="Satoshi"
```
Comment (KipandaJr) [inline views/institutional/components/feature/feature-header.tsx]: u cant put this props `fontFamily`

**Review by KipandaJr — CHANGES_REQUESTED** [review]
image

the `width` is diff in compare to fix


image
the space between image and text r so long

File: components/svg/svg.types.ts [hunk 9]
```diff
@@ -2,5 +2,5 @@ import { SVGAttributes } from 'react';
 
 export interface SVGProps extends SVGAttributes<SVGSVGElement> {
   maxWidth: string;
-  maxHeight: string;
+  maxHeight?: string;
```
Comment (KipandaJr) [inline components/svg/svg.types.ts]: Why u change this sir ?

File: views/institutional/components/feature/feature-card.tsx [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+        'center',
+        toRevert ? 'flex-start' : 'flex-end',
+      ]}
+    >
+      <Typography
+        display="flex"
+        justifyContent="center"
+        alignItems="center"
+        color="surface"
+        bg="#0053DB"
```
Comment (KipandaJr) [inline views/institutional/components/feature/feature-card.tsx:51]: this Colors dosent exist on `ui-kit`

File: views/institutional/components/feature/index.tsx [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+    />
+    <FeatureCard
+      number={4}
+      description="Create a DAO with one function call"
+      icon={
+        <MyMoveDaoSVG maxWidth="43.938rem" width="100%" maxHeight="28.438rem" />
+      }
+      toRevert
+      isCentered
+    />
```
Comment (KipandaJr) [inline views/institutional/components/feature/index.tsx]: Build one object to map this

File: public/img/features.webp
Comment (KipandaJr) [inline public/img/features.webp:1]: this image r necessary?

**Review by KipandaJr — CHANGES_REQUESTED** [review]
The mobile version are diff plz check this too

**Pull request merged**